src/jobs/test-nut.yml

```diff
@@ -45,12 +49,27 @@ steps:
   - checkout
   - when:
       condition:
-        not:
-          equal: ['', <<parameters.repo_tag>>]
+        and:
+          - equal: ['windows', <<parameters.os>>]
+          - not: <<parameters.repo_tag>>
+      steps:
+        - run:
+            name: Switch to tag v<<parameters.repo_tag>> or  <<parameters.npm_module_name>>@<<parameters.repo_tag>>
+            command: |
+              git checkout v<<parameters.repo_tag>>
```

Comment:
doesn't line 54 mean there is no repo_tag?  How can it be used here?

Reply:
@mshanemc Good catch!